lib/src/group_custom_button.dart

```diff
@@ -72,24 +74,50 @@ class GroupCustomButton extends StatelessWidget {
         borderRadius: borderRadius,
         boxShadow: _boxShadow,
       ),
-      child: ElevatedButton(
-        onPressed: onPressed,
-        style: ElevatedButton.styleFrom(
-          elevation: elevation ?? 0.0,
-          primary: _getBackgroundColor(theme),
-          shape: _buildShape(),
-          padding: (width != null || height != null) ? EdgeInsets.zero : null,
-          alignment: (width != null || height != null) ? alignment : null,
-        ),
-        child: Padding(
-          padding: textPadding,
-          child: Text(
-            text,
-            textAlign: textAlign,
-            style: _textStyle,
-          ),
-        ),
-      ),
+      child: !isDisable
+          ? ElevatedButton(
+              onPressed: onPressed,
+              style: ElevatedButton.styleFrom(
+                elevation: elevation ?? 0.0,
+                primary: _getBackgroundColor(theme),
+                shape: _buildShape(),
+                padding:
+                    (width != null || height != null) ? EdgeInsets.zero : null,
+                alignment: (width != null || height != null) ? alignment : null,
+              ),
+              child: Padding(
```

Comment:
This type of widgets build is huge and need in much place to code 
Please simplify this code